extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -239,9 +245,24 @@ function _checkIfCanAddMeasurementsToDisplaySet(
   }
 }
 
-function _measurementReferencesSOPInstanceUID(measurement, SOPInstanceUID) {
+function _measurementReferencesSOPInstanceUID(
+  measurement,
+  SOPInstanceUID,
+  frameNumber
+) {
   const { coords } = measurement;
 
+  // NOTE: The ReferencedFrameNumber can be multiple values according to the DICOM
+  //  Standard. But for now, we will support only one ReferenceFrameNumber.
+  const ReferencedFrameNumber =
+    (measurement.coords[0].ReferencedSOPSequence &&
+      measurement.coords[0].ReferencedSOPSequence[0] &&
+      measurement.coords[0].ReferencedSOPSequence[0].ReferencedFrameNumber) ||
```

Comment:
same

Reply:
fixed in a9d22efe494178b6cbbdaaf971700f4b9e1bf517